synthea-hiv/uploader/resources.py

```diff
@@ -0,0 +1,131 @@
+"""Resources that OpenMRS can upload. This class is only used for OpenMRS."""
+
+import random
+from typing import Dict, List
+import uuid
+
+import idgen
+
+
+class BaseResource:
+  """Base resource. Has a JSON, original_id, and potentially, a new_id."""
+
+  def __init__(self, json: Dict[str, str]):
+    self.json = json
```

Comment:
If these properties are meant to be accessed directly inside the class only, then please add `_` prefix to them.